File: LV1/1.2/qaly/qaly.cpp
#include <cstdio>

int main() {
    int n;
    double a, b, ans = 0;
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%lf %lf", &a, &b);
        ans += a * b;
    }
    printf("%lf\n", ans);
}

File: LV1/1.2/qaly/qaly.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut buff = stdin.lock().lines().map(|l| l.unwrap());
    buff.next();

    let mut ans: f64 = 0.0;
    for line in buff {
        let num: Vec<f64> = line
            .split_whitespace()
            .map(|n| n.parse().unwrap())
            .collect();
        ans += num[0] * num[1];
    }
    println!("{}", ans);
}
